django/db/backends/mysql/operations.py

```diff
@@ -173,6 +181,17 @@ def quote_name(self, name):
     def random_function_sql(self):
         return 'RAND()'
 
+    def return_insert_columns(self, fields):
+        if not fields:
```

Comment:
I would add a `return` when it's not supported (even if it shouldn't be called):
```python
if not self.connection.features.can_return_columns_from_insert:
    return
```

Reply:
I am not sure that's valuable? We don't do the same in any other operations that I can find - it shouldn't be called, and implicitly returning `None` would just break any caller anyway.

Reply:
So maybe a comment, it will make life easier (at least mine)
```
# MySQL and MariaDB < 10.5.0 don't support an INSERT...RETURNING statement.
```

Reply:
Fair enough, adding :)